sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/MessageSerializerResult.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp.implementation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/***
+ * This will hole additional data which can not be stored in AMQP Message.
+ * Example last  serial number.
+ */
+public class MessageSerializerResult<T> {
+    List<T> messages = null;
```

Comment:
Access modifier and using final instead. Consider setting Collections.emptyList instead.

Reply:
removed this class.